### DISCUSSION TITLE
Welcome to minted-export Discussions!

### DISCUSSION THREAD

**Discussion by wkarney** [wkarney/minted-export#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to quickly address any questions people might have regarding the minted-export tools. You're welcome to also provide any general feedback on your experience using the tools as well. We hope that you:
  * Ask questions or state issues that you're having implementing the tool.
  * Share answers / ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that everyone has different levels of comfort with the technology utilized; there is no such thing as a dumb question.

  To get started, comment below.

**Comment by meglee18** [comment]
Hello, I am going through your automated method and slightly confused. "2. Navigate into the repository you just cloned: cd minted-export"

I don't see anything named minted-export?

**Reply by wkarney** [reply]
The whole repository is called `minted-export`. In Step 1, you have cloned the repository to your local machine. You should now see a new folder in that directory on your local machine named `minted-export`. Step 2 is saying make sure you are in that directory (cd = change directory).
